whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -223,6 +223,18 @@ public Response searchDomains(
         return handleSearch(new String[]{"domain"}, name, request);
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, CONTENT_TYPE_RDAP_JSON})
+    @Path("/help")
+    public Response help(@Context final HttpServletRequest request) {
+
+        LOGGER.info("Request: {}", RestServiceHelper.getRequestURI(request));
```

Comment:
remove this logging before merging (or the console may be flooded with help request messages